app/views/AnalyticalFramework/Review/index.tsx

```diff
@@ -37,7 +39,7 @@ function Review(props: Props) {
     const secondaryTagging = secondaryTaggingFromProps;
 
     const [selectedSection, setSelectedSection] = useState<string | undefined>(
-        primaryTagging[0]?.clientId,
+        primaryTagging && primaryTagging[0]?.clientId,
```

Comment:
Any reason this was changed?

Reply:
Deep scan showed the null-check error in that line